backend/src/main/java/dev/tripdraw/application/AuthService.java

```diff
@@ -1,6 +1,7 @@
 package dev.tripdraw.application;
 
 import static dev.tripdraw.exception.member.MemberExceptionType.MEMBER_NOT_FOUND;
+import static dev.tripdraw.exception.member.MemberExceptionType.NICKNAME_CONFLICT;
```

Comment:
```suggestion
import static dev.tripdraw.exception.member.MemberExceptionType.DUPLICATE_NICKNAME;
```

어떨지요?
CONFLICT라는 뜻이 중복이라는 뜻으로 쓰이고 있나요? 제가 영어가 약해서..

Reply:
이름 좋네요!